packages/core/src/external/aws/lambda-logic/bundle-to-html.ts

```diff
@@ -972,27 +972,43 @@ type RenderCondition = {
   clinicalStatus: string;
 };
 
-function createConditionSection(conditions: Condition[]) {
+function createConditionSection(conditions: Condition[], encounter: Encounter[]) {
   if (!conditions) {
     return "";
   }
 
+  const conditionDateDict = getConditionDatesFromEncounters(encounter);
+
   const removeDuplicate = uniqWith(conditions, (a, b) => {
     const aDate = dayjs(a.onsetDateTime).format(ISO_DATE);
     const bDate = dayjs(b.onsetDateTime).format(ISO_DATE);
-    return aDate === bDate && a.code?.text === b.code?.text;
+    const aText = a.code?.text;
+    const bText = b.code?.text;
+
+    if (aText === undefined || bText === undefined) {
```

Comment:
We could use `==` instead of `===` in these situations, it will also return true if the value is `null`, essentially testing for `nullish` instead of just `undefined`.